app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,94 @@
+---
+name: Kong jq
+publisher: Kong Inc.
+versions: 0.1.x
+beta: true
+
+desc: Transform JSON objects included in API requests or responses using jq filters and programs.
```

Comment:
I introduced the term `filter` in the original version to denote the combination of a `program + options + source + target + conditions`, as previously you can add an arbitrary number of these. It's no longer relevant in this simpler version.

```suggestion
desc: Transform JSON objects included in API requests or responses using jq programs.
```

Reply:
will fix